functions/src/monitor/monitor.ts

```diff
@@ -51,6 +51,21 @@ export const ping: any = async (projectUid: string, monitorUid: string): Promise
   };
   pingResult.isValid = !!(pingResult.codeMatched && pingResult.textMatched);
 
+  // Updating monitor with latest ping information
+  monitor['latestPing'] = pingResult;
```

Comment:
Would this be better?

```suggestion
  monitor.latestPing = pingResult;
```